pkg/core/blockchain.go

```
@@ -1241,11 +1241,97 @@ func (bc *Blockchain) verifyTx(t *transaction.Transaction, block *block.Block) e
 		if bc.dao.IsDoubleClaim(claim) {
 			return errors.New("double claim")
 		}
+		if err := bc.verifyClaims(t); err != nil {
+			bc.log.Warn("error while verifying claims", zap.String("error", err.Error()))
```

`VerifyTx` should just return an error here.